### ISSUE TITLE
Console warning "Each child in a list should have a unique "key" prop"

### ISSUE THREAD

**Issue by tgfischer** [semoal/themeprovider-storybook#27]
**Description**
This warning is thrown in the console when using this addon

```
vendors~main.b9972167133ca606f27b.bundle.js:164613 Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <Styled(div)>. See https://fb.me/react-warning-keys for more information.
    in Unknown
    in Unknown
    in Unknown
    in Unknown
    in Unknown (created by Context.Consumer)
    in ManagerConsumer (created by _default)
    in _default (created by Layout)
    in div (created by Context.Consumer)
    in Styled(div) (created by Panel)
    in Panel (created by Layout)
    in div (created by Context.Consumer)
    in Styled(div) (created by Main)
    in div (created by Context.Consumer)
    in Styled(div) (created by Main)
    in Main (created by Layout)
    in Layout (created by Context.Consumer)
    in WithTheme(Layout)
    in Unknown
    in Unknown
    in div (created by Context.Consumer)
    in Styled(div)
    in Unknown
    in Unknown (created by SizeMeRenderer(Component))
    in SizeMeReferenceWrapper (created by SizeMeRenderer(Component))
    in SizeMeRenderer(Component) (created by SizeMe(Component))
    in SizeMe(Component) (created by Manager)
    in ThemeProvider (created by Manager)
    in Manager (created by Context.Consumer)
    in Location (created by QueryLocation)
    in QueryLocation (created by Root)
    in LocationProvider (created by Root)
    in HelmetProvider (created by Root)
    in Root
```

I'm using NodeJS `v12.14.1`, Storybook `v5.3.10`, and `themeprovider-storybook` `1.2.5`.

My `main.js` file looks like this. Taking out the `themeprovider-storybook` addon removes the warning

```
module.exports = {
  stories: ["../src/**/*.stories.(js|mdx)"],
  addons: [
    "@storybook/preset-create-react-app",
    "@storybook/addon-actions",
    "themeprovider-storybook",
    {
      name: "@storybook/addon-docs",
      options: {
        configureJSX: true,
        babelOptions: {},
        sourceLoaderOptions: null
      }
    }
  ]
};
```

**Comment by semoal** [comment]
I'm not sure this warning is caused by this library, but i'll take a look
Thanks

**Comment by semoal** [comment]
Confirmed the warning comes from our library.
It's fixed easily with a minimal change on `register.jsx`, so I'll deploy a new version with this change and the other ones soon.

**Comment by semoal** [comment]
Fixed in # [1.3.0](https://github.com/semoal/themeprovider-storybook/compare/v1.2.5...v1.3.0) (2020-02-05)


### Bug Fixes

* console warning fixed ([fd71494](https://github.com/semoal/themeprovider-storybook/commit/fd71494d8a2ce2d9cb1a88b5cc8d2693d45a2187))
* fixed stories are not displayed correctly in DocsPage ([9cdf279](https://github.com/semoal/themeprovider-storybook/commit/9cdf2793b994d02b54e8b5a4824932897be66a3b))
* fixed Storyshots are blank when using this addon ([4971f0b](https://github.com/semoal/themeprovider-storybook/commit/4971f0ba0958b58850efbcae76775ebabfe6c640))


### Features

* now, stores selected theme on local-storage ([aaa4593](https://github.com/semoal/themeprovider-storybook/commit/aaa459342a7dae84a23a04fb709aee0bb808cd96))
